README.md

```diff
@@ -1,68 +1,130 @@
 [![Build Status](https://travis-ci.org/finos/alloy.svg?branch=master)](https://travis-ci.org/finos/alloy)
```

Comment:
This should be removed, it's wrong (we dont have any Travis build)

Reply:
@maoo It was in there already, so I didn't change it. I'll do so now.

Reply:
See https://github.com/brooklynrob/alloy/blob/master/README.md to see the pending change.